### ISSUE TITLE
Digit recognition(0-9) using Logistic-Regression

### ISSUE THREAD

**Issue by Rohit-bisht-rise** [Jayshah6699/datascience-mashup#85]
Hello again, 
This is a multiclass classification problem 
> dataset Logistic-Regression: sklearn.dataset
There will be preprocessing and visualization of the data, and then tuning the model for the right accuracy
It will be achieved by changing various hyperparameters for the models, 
at last accuracy with various metric like, MSE, R2, Confusion matrix, 
@Jayshah6699 Please add the label and assign.

**Comment by Amit366** [comment]
@Jayshah6699 is this issue open??

**Comment by Rohit-bisht-rise** [comment]
@Amit366 yes, this issue is open, I'm busy right now with the college examination, if you want to work upon this issue you are welcome.
**Please keep a note of the followings:**
* Use handwritten digits, I was using from Kaggle [check here](https://www.kaggle.com/artgor/handwritten-digits), you are free to work on any dataset if you find
* Use Logistic Classification Model for classifying, 
* You need to convert the images into the matrix to make DataFrame

Please confirm if you want to work on this.
@Jayshah6699 if @Amit366 wants, then assigned to him

**Comment by Amit366** [comment]
Yes sir I want to work on this.

**Comment by Amit366** [comment]
![Screenshot (79)](https://user-images.githubusercontent.com/60662775/106116013-6fd27c80-6177-11eb-9a8b-382fdeaad9ef.png)

**Comment by Amit366** [comment]
@Jayshah6699 I am getting this error when using git add .

**Comment by Jayshah6699** [comment]
change a file name a little bit
